contributed_definitions/NXmpes.nxdl.xml

```diff
@@ -206,6 +243,10 @@
                         </enumeration>
                     </field>
                     <group type="NXdata" recommended="true">
+                        <!--
+                            TODO: Anders Frisk on proposal page
+                                It could be a lot of raw data from a detector. Shpould be optional.
+                        -->
                         <attribute name="signal">
                             <enumeration>
                                 <item value="raw"/>
```

Comment:
We can make it optional also, yes. More importantly, maybe we should make it an NXfloat in accordance to NXdetector